.github/ISSUE_TEMPLATE/2_bug_report_windows.yml

```diff
@@ -0,0 +1,106 @@
+name: "Report a Windows Issue"
+description: "Create a report for a runtime related Windows Issue"
+labels: ["os:win"]
+body:
+- type: textarea
+  attributes:
+      label: "Important Information"
+      description: >
+          - mpv version
+           - Windows Version
```

Comment:
Similar comment as before, might give users a hint how to produce this information. 

`ver` (cmd) or `[Environment]::OSVersion` PowerShell

Maybe we should add `uname -a` like output to `mpv --version`? To make it easier? But it is definitely not in the scope of this PR.

Reply:
Small note here:

`Environment]::OSVersion` does not include the "full" version of Windows, i.e. the build version.
Still, it should not matter in this case..